File: ReservationFrontend/src/models/enums/ReservationStatus.ts
export enum ReservationStatus {
  PENDING = "PENDING", //Customer created reservation,but not paid yet
  EXPIRED = "EXPIRED", //Customer created reservation,but not paid yet
  CANCELLED = "CANCELLED",
  CONFIRMED = "CONFIRMED", // Customer reserved a vehicle
  PICKED_UP = "PICKED_UP", // Customer picked up the vehicle
  DELIVERED = "DELIVERED", // Customer returned the vehicle
}

File: AnalyticsService/src/main/kotlin/com/rentalcarsystem/analyticsservice/enums/ReservationStatus.kt
package com.rentalcarsystem.analyticsservice.enums

enum class ReservationStatus {
    PENDING,    // Customer created reservation, but it has not been paid yet
    EXPIRED,    // Customer did not pay the reservation within the allowed time frame
    CANCELLED,  // Customer cancelled the reservation before it was set as expired
    CONFIRMED,  // Customer successfully paid and reserved a vehicle
    PICKED_UP,  // Customer picked up the vehicle
    DELIVERED,  // Customer returned the vehicle
}